### ISSUE TITLE
Support Coverify

### ISSUE THREAD

**Issue by stevepeak** [codecov/support#17]
> re @cbas, @yoshuawuyts

- [ ] example output coverage document
- [ ] create an example repo at [codecov/example-coverify](https://github.com/codecov/example-coverify)

**Comment by yoshuawuyts** [comment]
is there any way we can assist?

**Comment by cbas** [comment]
@stevepeak Here's some stuff to play around with:
https://gist.github.com/cbas/7e0bda3f8101b503465e

Sorry for the gist. GitHub doesn't seem to allow pull requests on blank repos.

**Comment by stevepeak** [comment]
@cbas thanks for the output example! That is a great start. I'm curious of the format of these reports. It looks like it only reports covered lines which I believe @yoshuawuyts informed me of this before.

Is there any other output methods for Coverfy? Can we produce a lcov/json/html outputs?

**Comment by yoshuawuyts** [comment]
to my knowledge that's the only format it outputs; I've read through the source a while back and couldn't find any way to make it output something else.

**Comment by stevepeak** [comment]
Could lcov or another format be added via a pull-request to substack/coverify? I have not checked out the project code yet. What options do we have?

**Comment by yoshuawuyts** [comment]
Yeah, that's an option. Not sure how easily it integrates, but knowing substack's projects they're fairly modular so it's probably feasible.

**Comment by cbas** [comment]
Substack's [covert](https://www.npmjs.com/package/covert) has an option to output JSON. Looking at the code it seems to get it from the stderr stream in coverify, but I'm not 100% clear on that. Maybe there is a way to extract JSON that way?
